dev/com.ibm.ws.rest.handler.validator.cloudant_fat/publish/servers/com.ibm.ws.rest.handler.validator.cloudant.fat/server.xml

```diff
@@ -0,0 +1,56 @@
+<!--
+    Copyright (c) 2019 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="new server">
+
+  <featureManager>
+    <feature>componenttest-1.0</feature>
+    <feature>configValidator-1.0</feature> <!-- TODO replace when functionality is enabled via auto-feature -->
+    <feature>cloudant-1.0</feature>
+    <feature>mpOpenApi-1.0</feature>
```

Comment:
does this feature need to be enabled? We should enable the minimal set of features so only the FAT auto detection doesn't run unnecessary FATs for future code changes

Reply:
Yep, the mpOpenApi feature is not needed, I'll remove it.